.github/workflows/release.yml

```diff
@@ -0,0 +1,30 @@
+---
+name: Release
+env:
+    EXODUS_API_KEY: ${{secrets.EXODUS_API_KEY}}
+on:
+  push:
+    tags:
+      - release-v*
+  workflow_dispatch:
+jobs:
+    release:
+        runs-on: ubuntu-latest
+        steps:
+        - name: Checkout the code ✅
+          uses: actions/checkout@v3
+        - name: Setup Java JDK 🔧
+          uses: actions/setup-java@v1
+          with:
+            java-version: 11
+        - name: Generate release build 🔧
+          run: ./gradlew assemble
+        - name: Execute exodus-standalone 🔎
+          uses: docker://exodusprivacy/exodus-standalone:v1.3.1
+          with:
+            args: /github/workspace/app/build/outputs/apk/release/app-release-signed.apk
+        - name: Upload APK 🚀
+          uses: actions/upload-artifact@v3
+          with:
+            name: app_release
+            path: app/build/outputs/apk/release/*.apk
```

Comment:
**suggestion**: If we know the exact path of the apk, can we put it instead of a wildcard?